docs/INSTALL.md

```diff
@@ -242,10 +242,11 @@ _macOS:_
 
 _Windows 10:_ no action needed, should work out of the box.
 
+Make sure the Studio has at least 1920Mb max heap size in "File > Settings... > Appearance & Behavior > System Settings > Memory Settings" _before_ opening the project, otherwise it might get stuck on "Updating indexes".
```

Comment:
True.
But the "File > Settings... >" menu in not available unless there is an open project :)
Its possible to set it differently via a "gear" menu icon in bottom left > Edit Custom VM Options...
and add there e.g. `-Xmx2560m`

Reply:
Yeap the entire menu is available after getting into the SDK Manager (just checked it on Linux), so it looks like the best option indeed!

Reply:
Updated in #2243